addons/website_sale/static/src/js/website_sale.js

```diff
@@ -37,8 +43,17 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
             placement: 'auto',
             template: '<div class="popover mycart-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>'
         });
+        window.addEventListener('visibilitychange', this._onVisibilityChange.bind(this));
+        this._updateCartQuantityText();
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    destroy() {
+        window.removeEventListener('visibilitychange', this._onVisibilityChange.bind(this));
```

Comment:
I feel like this would not really remove the event listener as calling bind would create a new object (bound function or whatever it would be).
I tested it quickly like this: 
```js
class Test {
    test() {
        console.log('test');
    }
}
const test = new Test();
const fnt1 = test.test.bind(test);
const fnt2 = test.test.bind(test);
fnt1 === fnt2;
false
```

Reply:
Indeed !! Thanks a lot !